addons/l10n_hu_edi/models/account_move.py

```diff
@@ -561,15 +574,15 @@ def _l10n_hu_edi_query_status_single_batch(self, connection):
                     'l10n_hu_edi_messages': {
                         'error_title': _('The invoice was sent to the NAV, but there was an error querying its status.'),
                         'errors': e.errors,
-                        'blocking_level': 'error_but_continue',
```

Comment:
do we still have error_but_continue left?

Reply:
No :smile: 
I had misunderstood the use of `error_but_continue` in the Send & Print: it's used in cases where the EDI has some kind of error which prevents an official tax invoice from being generated (e.g. if a QR code is missing) but we should still generate a proforma PDF.

I don't think it is applicable to Hungary, since the normal PDF can always be generated (it does not require anything retrieved from the government). Therefore, if there is an actual error (invoice rejected), we block the user with a UserError. In all other cases, we just provide a message in the banner.

I appreciate this is not the best in the case where NAV is slow, so the Send and Print returns but we are still waiting for a response from the government (in this case, we put this in the banner, but don't block the user, the PDF is generated and the e-mail is sent to the customer etc.) This might not be the best since the invoice might still be rejected later. I'll have a chat with LAS tomorrow to see whether we can do something to the Send and Print to handle this case better (which is also needed in other localizations where the sending is in 2 steps e.g. Peru).